agent/tool-scripts/base-tool

```diff
@@ -711,19 +710,21 @@ start)
 
 	if [[ ! -z "${options}" ]]; then
 		# Record any options for later post-processing use.
-		echo "${options}" > ${tool_output_dir}/${tool_name}.options
+		echo "${options}" > ./${tool_name}.options
 	fi
 
-	tool_cmd_file="${tool_output_dir}/${tool_name}.cmd"
+	tool_cmd_file="./${tool_name}.cmd"
 	if [[ -z "${gopath}" ]]; then
 		if [[ "${tool_package_name}" == "golang" ]]; then
 			printf -- "%s: golang based tools require the --gopath option\n" "${tool_name}" >&2
+			popd > /dev/null
```

Comment:
Does it make a difference doing a `popd` before exiting? There are a few more instances on ll. 727 and 738 as well.

Reply:
I don't think so, but wanted to be make sure we explicitly left the context.  I am not in favor of assuming things work a certain way.